src/schema/causality_jwt.ts

```diff
@@ -102,7 +102,12 @@ const CausalityJWT: GraphQLFieldConfig<void, ResolverContext> = {
 
           if (sale.partner) {
             return mePartnersLoader({ "partner_ids[]": sale.partner._id }).then(
-              () => {
+              mePartners => {
+                // Check if current user has access to partner running the sale
+                if (mePartners.length === 0) {
+                  throw new Error("Unauthorized to be operator")
```

Comment:
Given this, do we need the `throw new Error` statement below? Or maybe we should change the message since it seems like it'd get triggered if we tried to access partner operator for a sale that has no partner?

Reply:
Prediction directly depends on [the error message](https://github.com/artsy/prediction/blob/ca0c0e4da4233078ba9d8c58e425a26bfd9feeee/src/server/helpers.ts#L67) so it would require a few more changes if we want to be more explicit (any other error messages than `"Unauthorized to be operator"` would result in 500 right now), but it shouldn't be too difficult to distinguish two cases.